src/components/organisms/OcModal/OcModal.vue

```diff
@@ -34,6 +34,14 @@
             @keydown.enter="confirm"
           />
           <p v-else key="modal-message" class="oc-modal-body-message" v-text="message" />
+          <div v-if="checkboxLabel" class="oc-modal-body-actions oc-flex oc-flex-left">
+            <oc-checkbox
+              v-model="checkboxValue"
+              size="medium"
+              :label="checkboxLabel"
+              :aria-label="checkboxLabel"
```

Comment:
if aria-label and accessible content (in this case: `label`) are the same you don't need the aria-label. You can drop it completely.